src/dokusan/generators.py

```diff
@@ -31,20 +31,20 @@ def _random_initial_cells(box_size: BoxSize) -> List[Cell]:
     size = box_size.width * box_size.length
     all_values = set(range(1, size + 1))
 
-    values = random.sample(all_values, k=size)
+    values = random.sample(sorted(all_values), k=size)
```

Comment:
We don't need the sequence to be sorted, so we can just convert it to list
```suggestion
    values = random.sample(list(all_values), k=size)
```